8/SiloHost/src/Program.cs

```diff
@@ -22,10 +19,10 @@ public static Task Main()
                 .UseOrleans(siloBuilder =>
                 {
                     IEnvironmentVariables environmentVariablesService = new EnvironmentVariables();
+
                     siloBuilder.UseLinuxEnvironmentStatistics();
-                    siloBuilder.ConfigureDashboardOptions(environmentVariablesService);
-                    //Register silo with dynamo cluster
-                    siloBuilder.ConfigureDynamoClusterOptions(environmentVariablesService);
+                    siloBuilder.ConfigureDashboardOptions(environmentVariablesService.GetDashboardPort());
+                    siloBuilder.ConfigureDynamoDbClusteringOptions(environmentVariablesService.GetMembershipTableName(), environmentVariablesService.GetAwsRegion());
```

Comment:
I modified these two for two reasons:

* I want to limit the number of places where we do I/O and I consider env var reading an I/O operation.
* Not letting the helper rely on `IEnvironmentVariables` will make it truly dependency-free, so engineers will have an easier time experimenting/mixing-and-matching their preferred settings.

Reply:
The remaining invocations of the helper should also be updated, just didn't have time to do it myself today. I should have some time on Monday if you'd like my help.